### PR TITLE
[Don't merge] [DNS] Record comments and tags

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/dns/manage-dns-records/how-to/import-and-export.md [hunk 1]
```diff
@@ -53,3 +55,36 @@ To export records using the dashboard:
...
 To export records using the API, send a [GET request](https://developers.cloudflare.com/api/operations/dns-records-for-a-zone-export-dns-records).
+
+---
+
+## DNS record attributes
+
+When exporting or importing a zone file, Cloudflare formats [comments and tags](/dns/manage-dns-records/reference/record-attributes/) using the following structure, appending the attributes as inline comment using the `;` character after each record in accordance with [RFC 1035 section 5](https://datatracker.ietf.org/doc/html/rfc1035#section-5-1):
+
+| Combination | Description |
+| --- | --- | --- |
```
Comment (ranbel) [inline content/dns/manage-dns-records/how-to/import-and-export.md]:
```suggestion
| --- | --- |
```

File: content/dns/manage-dns-records/how-to/import-and-export.md [hunk 2]
```diff
@@ -53,3 +55,36 @@ To export records using the dashboard:
...
+| **Only tags** | Tag names contain a [small set](/dns/manage-dns-records/reference/record-attributes/#tags) of characters.<br/><br/>Additionally, tag values must be contained by a double quote (`"`) if they contain `"`, `=`, `,`, or `\`.<br/><br/>A tag with an empty value can be represented either as `my-tag-name:""`, `my-tag-name:`, or `my-tag-name`. | 
+| **Only a comment** | Comments have [fewer limitations](/dns/manage-dns-records/reference/record-attributes/#comments) on characters, meaning that the comment is included verbatim.<br/><br/>If the comment includes the string `cf_tags=`, you need to include an additional ` cf_tags=` at the end of the line. |
+| **Comment and tags** | The zone file comment would be of the form ; `<comment>` cf_tags=`<tags>`, as described above. Note the added space character before `cf_tags=`. |
```
Comment (ranbel) [inline content/dns/manage-dns-records/how-to/import-and-export.md:69]: Is the semicolon supposed to be there?

**Review by ranbel — APPROVED** [review]

File: content/dns/manage-dns-records/reference/record-attributes.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+You can also add or edit attributes by [exporting and re-importing](/dns/manage-dns-records/how-to/import-and-export/#dns-record-attributes) your records.
+
+---
+
+## Reference
+
+### Comments
+
+Comments are treated as [graphic Unicode characters](https://en.wikipedia.org/wiki/Graphic_character), meaning that they are case sensitive and do not have any character limitations. However, comments do not support newline (`\n`) or carriage return (`\r`) characters.
```
Comment (ghost) [inline content/dns/manage-dns-records/reference/record-attributes.md:55]: ... or control characters (not sure if it matters)

File: content/dns/manage-dns-records/how-to/import-and-export.md [hunk 4]
```diff
@@ -53,3 +55,36 @@ To export records using the dashboard:
...
+
+---
+
+## DNS record attributes
+
+When exporting or importing a zone file, Cloudflare formats [comments and tags](/dns/manage-dns-records/reference/record-attributes/) using the following structure, appending the attributes as inline comment using the `;` character after each record in accordance with [RFC 1035 section 5](https://datatracker.ietf.org/doc/html/rfc1035#section-5-1):
+
+| Combination | Description |
+| --- | --- | --- |
+| **Only tags** | Tag names contain a [small set](/dns/manage-dns-records/reference/record-attributes/#tags) of characters.<br/><br/>Additionally, tag values must be contained by a double quote (`"`) if they contain `"`, `=`, `,`, or `\`.<br/><br/>A tag with an empty value can be represented either as `my-tag-name:""`, `my-tag-name:`, or `my-tag-name`. |
```
Comment (ghost) [inline content/dns/manage-dns-records/how-to/import-and-export.md]: When enclosed in double quotes, tag values are represented as a JSON string (so e.g. any other quotes within the value itself could be escaped as `\"`)

**Pull request merged**